lib/frame_repository.rb

```diff
@@ -0,0 +1,21 @@
+#require 'frame'
+
+class FrameRepository
```

Comment:
The term `Repository` typically indicates storage of some kind, and is used when writing classes that interact with a database. `Frame` is a good name for this class, as each _instance_ represents one frame of the game.